horreum-backend/src/main/resources/db/changeLog.xml

```diff
@@ -3102,6 +3102,15 @@
             CREATE TRIGGER ds_after_schema_update AFTER INSERT OR UPDATE OF uri ON schema FOR EACH ROW EXECUTE FUNCTION ds_after_schema_update_func();
         </sql>
     </changeSet>
+    <changeSet id="70" author="dupliaka">
+        <addColumn tableName="tablereport_data">
+            <column name="baseline" type="boolean"/>
```

Comment:
For boolean it would be better to have a non-null constraint. In changeset you need to do that by defining it with default `false` as well and then immediately `dropDefault`.